lex.go

```diff
@@ -162,8 +163,9 @@ func (l *lexer) nextItem() item {
 }
 
 // lex creates a new scanner for the input string.
-func lex(input string) *lexer {
+func lex(input string, preserveFormatting bool) *lexer {
```

Comment:
Since #37 wants to add another config parameter to the lexer I think we should move the `go l.run()` call to the `parse` function and refactor this function as follows:

```go
func lex(input string) *lexer {
	return &lexer{
		input: input,
		items: make(chan item),
		runes: make([]rune, 0, 32),
	}
}
```

Then `parse` looks like this:

```go
func parse(input string) (properties *Properties, err error) {
	l := lex(input)
	go l.run()
	p := &parser{lex: l}
...
}
```

This way it is possible to pass additional configuration to the lexer without adding them as function arguments.

Reply:
Done.